website/docs/user-group/user.md

```diff
@@ -20,6 +20,20 @@ Optional flag, when set to false, Tokens created by the user will not expire.
 
 See [Troubleshooting access problems](../troubleshooting/access.md), when set, the user gets a more detailed explanation of access decisions.
 
+### `additionalHeaders`:
+
```

Comment:
Could you make it a bit clearer that this field is only used with the proxy provider?

For example
```
:::info
This field is only used by the Proxy Provider.
:::
```